tests/unit/nupic/engine/network_test.py

```diff
@@ -280,8 +281,6 @@ def testOneRegionNetwork(self):
       print "Time for 1M getParameter calls: %.2f seconds" % (t2 - t1)
 
 
-  @unittest.skipIf(sys.platform.lower().startswith("win"),
```

Comment:
Don't know why this was tagged with skip for win, the test passes on windows without the skip.